src/transformers/cache_utils.py

```diff
@@ -395,21 +395,21 @@ def from_legacy_cache(cls, past_key_values: Optional[Tuple[Tuple[torch.FloatTens
                 cache.update(key_states, value_states, layer_idx)
         return cache
 
-    def crop(self, maximum_length: int):
-        """Crop the past key values up to a new `maximum_length` in terms of tokens. `maximum_length` can also be
-        negative to remove `maximum_length` tokens. This is used in assisted decoding and contrastive search."""
+    def crop(self, max_length: int):
```

Comment:
What's the reason for changing the name? 

max_length seems more consistent with the rest of the repo, but wondering if there's another reason?

Reply:
none, just consistency 🤗 It wasn't caught in a former PR, fixing it now before the function gets released :)